chrome/content/zotero/xpcom/schema.js

```diff
@@ -1469,7 +1469,7 @@ Zotero.Schema = new function(){
 			});
 			yield _updateDBVersion('compatibility', _maxCompatibility);
 			
-			var sql = "INSERT INTO libraries (libraryID, libraryType, editable, filesEditable) "
+			var sql = "INSERT INTO libraries (libraryID, type, editable, filesEditable) "
 				+ "VALUES "
 				+ "(?, 'user', 1, 1), "
 				+ "(4, 'publications', 1, 1)"
```

Comment:
It's for consistency with migrateUserDataSchema, where we use 1 and 4 (known unused ids, coming from the global libraryIDs in 4.0) for upgrading users. So, actually, the '?' for 'user' should just be 1 here.